utils/label_map_util.py

```diff
@@ -19,26 +19,25 @@
 
 import tensorflow as tf
 from google.protobuf import text_format
-# from object_detection.protos import string_int_label_map_pb2
 from protos import string_int_label_map_pb2
 
 
 def _validate_label_map(label_map):
-  """Checks if a label map is valid.
+    """Checks if a label map is valid.
 
   Args:
```

Comment:
Also indent these docstring lines?

Reply:
Hmmm. I guess my IDE didn't do that automatically